src/client/components/login-form.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from "react";
+class LoginForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: "",
+      password: "",
+      result: ""
+    };
+  }
+
+  handleChange = event => {
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+  };
+  handleSubmit = event => {
+    event.preventDefault();
+
+    fetch("http://localhost:3000/api/auth/login/ ", {
```

Comment:
This will only work as long as we run it locally, we should possibly keep this as a config or environment variable.
We could have a `ROOT_URL` that depends on `process.env` so we have one when running in development "mode" and another when running in production

Reply:
Hi Niels, Yes that's correct. Can you please provide the Development URL / Production Url(if available) so that I will try.

Reply:
Yeah, I will have to find that 👍